src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableFieldWriter.class.php

```diff
@@ -32,6 +33,21 @@ class TCMSTableFieldWriter extends TCMSTableEditor
 
     protected $_sqlFieldName = null;
 
+    /**
+     * @var array|null
+     */
+    private $oldData;
+
+    /**
+     * @var string|null
+     */
+    private $oldTableName;
+
+    /**
+     * @var bool
+     */
+    private $isChangeFromTable = false;
```

Comment:
I think this variable name could be improved. What does it really want to say?

Reply:
Sounds good.

Reply:
ok